pom.xml

```diff
@@ -43,6 +43,8 @@
     <properties>
         <skip.compile>false</skip.compile>
         <version.org.wildfly.checkstyle-config>1.0.5.Final</version.org.wildfly.checkstyle-config>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
True, but in this case I want to be explicit due to the mixed support for 8 and 9 that the main code base is Java 8 only, even though it includes 9.  Also this will help if/when we bump the default to 9 at some future date, I suppose.